requirements/extras/pymemcache.txt

```diff
@@ -1 +1 @@
-python-memcached==1.59
+python-memcached==1.61
```

Comment:
Revert this change which causes test failures and allow the other changes to be merged.
```suggestion
python-memcached==1.59
```